tests/unittests/functions/report/test_firewatch_functions_report_get_issue_labels.py

```diff
@@ -31,6 +32,16 @@ def test_get_issue_labels_with_additional_labels(self):
         assert "firewatch" in labels
         assert "additional-label-1" and "additional-label-2" in labels
 
+    def test_get_issue_labels_with_additional_labels_file(self):
+        labels = self.report._get_issue_labels(
+            job_name=self.job.name,
+            step_name="test-step-name",
+            type="test_failure",
+            jira_additional_labels=[],
+            jira_additional_labels_filepath=helpers._get_additional_labels_filepath(),
+        )
+        assert "test-1" and "test-1" in labels
```

Comment:
I may be missing here something but why assert twice `test-1`

Reply:
You're right, that was a typo. Thanks for catching :)